doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -281,6 +281,8 @@ Matter Bridge
 
 * Updated by enabling Link Time Optimization (LTO) by default for the ``release`` configuration.
 * Removed support for the nRF54H20 devices.
+* Enabled Link Time Optimization (LTO) by default for the ``release`` configuration.
+* Enabled the default storage of the Wi-Fi firmware patch in external flash.
```

Comment:
Suggesting to reorganize the items like this:

* Updated:

  * By enabling Link Time Optimization...
  * By enabling the default storage...

* Removed support for the nRF54H20 devices.